app/services/account-service.js

```diff
@@ -306,6 +306,22 @@ const authenticate = async (email, password) => {
   }
   const isUser = await bcrypt.compare(password, user.passwordHash);
   if (isUser) {
+    // assign role on JWT; default to least privilege
+    let role = [];
```

Comment:
@entrotech updated here to build up a list of roles and concat into a single `.role` string on the user.